### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by flyz1** [d-ailin/GDN#7]
Hello, I want to run your code on SWaT and WADI datasets, but how can I get train.csv, test.csv, and list.txt? Is WADI_14days_new.csv in the WADI dataset the original training set (WADI_attackdataLABLE.csv is the original test set)?  Then it is processed by process() in TimeDataset.py to become the training data set used in your article?
Looking forward to your reply, thank you very much!

**Comment by UniqueRoHo** [comment]
It's also my question!
